src/pybind/mgr/dashboard/services/cephx.py

```diff
@@ -1,28 +0,0 @@
-# -*- coding: utf-8 -*-
-
-from .ceph_service import CephService
-
-
-class CephX(object):
-    @classmethod
-    def _entities_map(cls, entity_type=None):
-        auth_dump = CephService.send_command("mon", "auth list")
```

Comment:
Any clue about why we are deleting this file? I even didn't know about it, but it might be useful for any future cephx work...

Reply:
I think this was just used for cephx client endpoint and as tht endpoint is removed we decided to remove this file as well. But if we want to keep it for future use I don't mind.